ufo/user_test.py

```diff
@@ -70,6 +90,167 @@ def testListUsersHandler(self, mock_get_all):
       details_link = flask.url_for('user_details', user_id=mock_users[x].id)
       self.assertEquals(details_link in user_list_output, True)
 
+  @patch.object(user, '_RenderUserAdd')
+  def testAddUsersGetHandler(self, mock_render):
+    """Test the add users get handler returns _RenderUserAdd's result."""
+    return_text = '<html>something here </html>'
+    mock_render.return_value = return_text
```

Comment:
nit: If I understand Jonathan's thoughts correctly, this should render a real template instead of a mock?  But let's see what he thinks. 
@jpevarnek